wiki/en/Tips-Tricks-More.md

```diff
@@ -25,7 +25,7 @@ Jamulus user [Chris Rimple](https://sourceforge.net/u/chrisrimple/profile/) has
 
 ###  Have a undisturbed session on any Server
 
-You can have a "private", undisturbed session with other people on any Server, including publically listed Servers, by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
+You can have a "private", undisturbed session with other people on any Server, including those listed on a Directory, by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
```

Comment:
Unqualified "Directory" is wrong here.  It's nothing to do with whether it's listed on a directory or not.  In fact, the whole clause is redundant - "any Server" means just that.  The rest is just confusing as it doesn't make a difference to most people.

The bracketed text should be a separate sentence, at least.  Perhaps a separate paragraph with "**NOTE:**" at the front...

Reply:
I think the "confusing" rest should still be there - at least to explain that they are *not* really private, but are just not disturbed by others.

Reply:
If nobody knows the address of your server, then that might be good enough "private" to most people :-) But I suppose say:

_"Jamulus doesn't have any means to password-protect or otherwise control access to Servers. But you can have a "private", undisturbed session with other people on any Server, ..."_

Reply:
Get rid of private and just say undisturbed ;-)

Reply:
OK that works too.